src/products/firefox_android/testing.md

```diff
@@ -1,34 +1,22 @@
 # Testing Firefox for Android
 
-## Testing on single-locale builds
+## Testing on Nightly builds
 
-Once you have localized [all the strings needed to complete the Firefox for Android project](how_to_localize.md), and your language is now up on single-locale builds, you should start testing! This is the moment to engage as much people as possible: have them use and check-out these builds regularly. The more eyes on them, the better.
-
-First, authorize 3rd party app downloads from your Android phone’s settings. From your phone, download your locale from https://ftp.mozilla.org/pub/mobile/nightly/latest-mozilla-central-android-api-16-l10n/ by clicking on the most recent link with your locale code in it (Note: you may want to scroll down the page to make sure you are choosing the most recent build!)
-
-Then, choose the .apk link out of the 3 possible options listed. Download should start and once the app is installed, you should be able to test your locale.
+Once you have localized [all the strings needed to complete the Firefox for Android project](how_to_localize.md), and your language is now up on [Nightly builds](https://play.google.com/store/apps/details?id=org.mozilla.fenix), you should start testing! This is the moment to engage as much people as possible: have them use and check-out these builds regularly. The more eyes on them, the better.
```

Comment:
I know they're carry over, but I have dumb English questions: 
* Much people or many people?
* I think "check out", since this is a verb.

Reply:
Ha, I was wrong in both cases ;)